pkgs/servers/x11/xorg/default.nix

```diff
@@ -1779,11 +1779,11 @@ self: with self; {
   # THIS IS A GENERATED FILE.  DO NOT EDIT!
```

Comment:
Can we move this to another PR to be merged faster?

Reply:
> Can we move this to another PR to be merged faster?

the pthread update is already in staging, I just picked it here because it has a different tag format.